### PR TITLE
Support future changes to galaxy_ng's namespace serializer.

### PR DESCRIPTION
**Pull request by jctanner**
https://issues.redhat.com/browse/AAH-2721
https://github.com/pulp/pulp_ansible/issues/1598

### CODE REVIEW

File: pulp_ansible/app/tasks/collections.py [hunk 1]
```diff
@@ -685,8 +685,9 @@ async def _add_namespace(self, name, namespace_sha):
             else:
                 namespace["links"] = dict()
 
-            for key in ("pulp_href", "groups", "id", "related_fields"):
-                namespace.pop(key, None)
+            for key in ("pulp_href", "groups", "id", "related_fields", "users"):
+                if key in namespace:
+                    namespace.pop(key, None)
```
Comment (mdellweg) [inline pulp_ansible/app/tasks/collections.py:690]: There's no need for the extra if here.

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
Would this warrant for a changelog? Do we need an issue?

**Comment by jctanner** [comment]
I've synced with galaxy_ng from 4.4 up to 4.8 against a local instance of the new galaxy_ng code and none of them displayed the issue.

**Comment by jctanner** [comment]
now depends on https://github.com/pulp/pulp_ansible/pull/1599

**Comment by mdellweg** [comment]
I guess you'd need to rebase on that change once more.

Feel free to merge when done.

**Pull request merged**